### ISSUE TITLE
mnist：diverged

### ISSUE THREAD

**Issue by zenglian** [lisc55/InfoGAN#1]
0it [00:00, ?it/s][21:44:10]	036	Generator: nan	Discriminator: nan	Info: nan

**Comment by wangchang327** [comment]
I ran the traning again and it converged well. I wonder whether you tuned the hyperparameters or not, so plz give me more information.

However, NaN could be possible if loss of the generator becomes too large (if you have trained it for a very long time), and this is when you should stop training.

**Comment by zenglian** [comment]
I ran the same again and it converged this time. But the result is not good as yours after 100 epochs. Will investigate a bit more. 
![random](https://user-images.githubusercontent.com/5268434/75092685-37b0da00-55b5-11ea-98ef-24acf58e33bd.png)

**Comment by wangchang327** [comment]
That looks weird. My result last night was largely the same as previous ones. I guess you should test some models from early epochs like the 50<sup>th</sup>.
